src/main/scala/ai/privado/languageEngine/java/processor/JavaProcessor.scala

```diff
@@ -107,6 +110,10 @@ object JavaProcessor {
           println(s"${Calendar.getInstance().getTime} - Brewing result...")
           MetricHandler.setScanStatus(true)
           // Exporting Results
+          if (ScanProcessor.config.generateAuditReport) {
```

Comment:
@ankit-privado Export the AuditReport and the below IntermediateFileExport after we have exported the .privado json

Reply:
As the exporter is return `Either[String, Unit]` match and use the case Left and right as done for Intermediate File exporter

Reply:
Yes sure @khemrajrathore